code/lib/codemod/src/index.ts

```diff
@@ -82,7 +79,7 @@ export async function runCodemod(
     const result = spawnSync(
       'node',
       [
-        require.resolve('jscodeshift/bin/jscodeshift'),
+        join(resolvePackageDir('jscodeshift'), 'bin', 'jscodeshift'),
```

Comment:
Doesn't this need the `.js` extension at the end, now that we're not relying on Node's resolution to find the bin file, but just doing manual path construction?

Sorry for being blunt, but have you tested that this works?